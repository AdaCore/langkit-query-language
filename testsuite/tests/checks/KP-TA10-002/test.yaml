driver: 'checker'
rule_name: KP_TA10_002
project: 'prj.gpr'
