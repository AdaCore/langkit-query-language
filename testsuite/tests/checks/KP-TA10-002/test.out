p.ads:17:8: rule violation: possible occurrence of KP TA10-002
17 |    is (case B is when others => Self.Source_Files.Iterate);  --  FLAG
   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

p.ads:21:8: rule violation: possible occurrence of KP TA10-002
21 |    is (if B then raise Program_Error else Self.Source_Files.Iterate);  --  FLAG
   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

